calico/felix/test/test_config.py

```diff
@@ -660,3 +664,14 @@ def test_drop_valid(self):
             }
             config = load_config("felix_missing.cfg", host_dict=cfg_dict)
             self.assertEqual(config.ACTION_ON_DROP, value)
+
+    def test_interface_prefix(self):
+        cfg_dict = {"InterfacePrefix": "foo"}
+        config = load_config("felix_interface_prefix.cfg",
+                             host_dict=cfg_dict)
+        self.assertEqual(config.IFACE_PREFIX, ['foo'])
+
+        cfg_dict = {"InterfacePrefix": "foo,bar"}
+        config = load_config("felix_interface_prefix.cfg",
+                             host_dict=cfg_dict)
+        self.assertEqual(config.IFACE_PREFIX, ['foo', 'bar'])
```

Comment:
It would be nice to have pep8 / flake8 / hacking checks integrated as a part of CI process (as a tox chain for example). To make it possible it will require lots of small changes in project files, but it can be done as divided into small chunks step by step process to improve code style and after it's done make pep8 as a part of CI.

 What do you think about this?